Source/WebCore/platform/graphics/GraphicsLayer.cpp

```diff
@@ -444,6 +444,14 @@ void GraphicsLayer::setDrawsContent(bool b)
     m_drawsContent = b;
 }
 
+#if HAVE(HDR_SUPPORT)
+void GraphicsLayer::setDrawsHDRContent(bool b)
+{
+    ASSERT_IMPLIES(m_type == Type::Structural, false);
```

Comment:
Why not `ASSERT(m_type != Type::Structural)` or `ASSERT_IMPLIES(m_type == Type::Structural, !b)` ?

Reply:
This assertion was changed to `ASSERT(m_type != Type::Structural)`. I just copied it from `GraphicsLayer::setDrawsContent()`. I changed this assertion only here but I will clean the rest of assertions in a separate PR.